mithril-signer/src/services/epoch_service.rs

```diff
@@ -308,6 +308,51 @@ impl SignedEntityConfigProvider for SignerSignedEntityConfigProvider {
     }
 }
 
+#[cfg(test)]
+impl MithrilEpochService {
+    /// `TEST ONLY` - Create a new instance of the service using dumb dependencies.
```

Comment:
I think you can remove all the `TEST ONLY` comments since the functions are implemented with the `#[cfg(test)]` attribute.